.gitlab-ci-build-openmr.yml

```diff
@@ -69,6 +70,31 @@ retag dev to production:
   only:
     - production
 
+# Hit the testing webhook to replace the image
+deploy testing:
+  stage: deploy
+  image: alpine
+  script:
+    - apk add --update curl jq
+    - |
+      echo "Getting list of services"
+      services=$(curl --no-progress-meter --connect-timeout 5 --retry 5 --request GET "${TESTING_SWARMPIT_URL}api/services" \
+        --header 'Accept: application/json' \
+        --header "Authorization: Bearer ${PIT_AUTHORIZATION_TESTING}" \
```

Comment:
If I get this right, this should come from swarmpit, correct?

Reply:
Yes you are correct.